.github/actions/abort-release-candidate-v1/action.yml

```diff
@@ -0,0 +1,106 @@
+name: abort-release-candidate
+description: 'A GitHub Action to abort a release candidate'
+
+inputs:
+  token:
+    description: 'Token used for the GH CLI to close and move issue(s) to the correct column. See README for required permissions'
+    required: true
+  base:
+    description: 'The branch the release candidate will be merged into'
+    required: true
+  docs-repo:
+    description: 'The docs repository where the issue for the release candidate is located'
+    default: 'null'
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        # Fetch all history
+        fetch-depth: 0
+
+    - name: Checking out release branch and get release candidate version
+      shell: bash
+      id: get-release-candidate-version
+      run: |
+        git checkout release
+
+        # Get the version number from the prepare_release shell script and set it as an output
+        # Monorepos have a lerna.json file that contains the updated version number
+        # Non-monorepos, or non-npm repos, have a package.json file that contains the updated version number
+        if [ -f "lerna.json" ]; then
+          echo "release_candidate_version=$(cat lerna.json | jq -r '.version')" >> $GITHUB_OUTPUT
+        else
+          echo "release_candidate_version=$(cat package.json | jq -r '.version')" >> $GITHUB_OUTPUT
+        fi
+
+    - name: Get release candidate issue url
+      uses: dequelabs/axe-api-team-public/.github/actions/get-release-issue-v1@main
+      id: get-release-issue
+      with:
+        version: ${{ steps.get-release-candidate-version.outputs.release_candidate_version }}
+      env:
+        GH_TOKEN: ${{ inputs.token }}
+
+    - name: Close release candidate issue as `Not Planned`
```

Comment:
Since not having a release issue returns a warning, do you know if gh throws an error if you try to close an already closed issue or an issue that doesn't exist? Might need to guard this action behind an `if` checking that the previous action returned an url.

Reply:
Confirmed if the previous step throws the warning the output would be `''` and the `gh` command does throw. Added guard for that.